go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -125,7 +125,7 @@ func (oc *Controller) addEgressFirewall(egressFirewall *egressfirewallapi.Egress
 		return fmt.Errorf("unable to add egress firewall policy, namespace: %s has no address set", egressFirewall.Namespace)
 	}
 
-	err = oc.addLogicalRouterPolicyToClusterRouter(nsInfo.addressSet.GetIPv4HashName(), nsInfo.addressSet.GetIPv6HashName(), egressFirewall.Namespace, egressFirewallStartPriorityInt)
+	err = oc.addEgressFirewallRules(nsInfo.addressSet.GetIPv4HashName(), nsInfo.addressSet.GetIPv6HashName(), egressFirewall.Namespace, egressFirewallStartPriorityInt)
```

Comment:
egressFirewallStartPriorityInt comes from a list of constants for logical_router policies. Is there any issues with using the same priority on the join_switch?

Reply:
as long as the priority does not conflict with the existing acls, I don't think there is any problem.